geopandas/io/tests/test_pickle.py

```diff
@@ -80,7 +83,10 @@ def _create_gdf():
     )
 
 
-@pytest.mark.skipif(not compat.HAS_PYGEOS, reason="requires pygeos to test #1745")
+@pytest.mark.skipif(
+    not compat.HAS_PYGEOS or compat.USE_SHAPELY_20,
```

Comment:
```suggestion
    not (compat.USE_PYGEOS or compat.USE_SHAPELY_20),
```

(I think you want `USE_PYGEOS` instead of `HAS_PYGEOS` here)

Reply:
In this specific test, I think  `HAS_PYGEOS` is actually fine, since the test itself switches between enabling/disabling pygeos (and thus can always be run if pygeos is available). 

And actually, this test should work in both cases of shapely 1.8 or shapely 2.0, so I can just remove the addition here)